DirectProgramming/DPC++FPGA/ReferenceDesigns/qri/sample.json

```diff
@@ -0,0 +1,69 @@
+{
+  "guid": "A42A1772-2448-4f67-AB28-4080FE47F9C1",
+  "name": "QRI",
+  "categories": ["Toolkit/oneAPI Direct Programming/DPC++ FPGA/Reference Designs"],
+  "description": "Reference design demonstrating high-performance QR-based matrix inversion (QRI) of real and complex matrices on a Intel® FPGA",
+  "toolchain": ["dpcpp"],
+  "os": ["linux", "windows"],
+  "builder": ["ide", "cmake"],
+  "targetDevice": ["FPGA"],
+  "languages": [{"cpp":{}}],
+  "ciTests": {
+    "linux": [
+      {
+        "id": "fpga_emu",
+        "env": [
+          "export CL_CONFIG_CPU_FORCE_PRIVATE_MEM_SIZE=32MB"
+        ],
+        "steps": [
+          "dpcpp --version",
+          "cd ../..",
```

Comment:
No need for the direcory changing in Linux, please only do this in Windows.  Applies to reports test below as well.